### ISSUE TITLE
Xcode 12 GM SPM Firebase Analytics Run Error - Failed to load Info.plist from bundle

### ISSUE THREAD

**Issue by harrygt** [firebase/firebase-ios-sdk#6473]
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 12 GM (12A7209) running on iOS 14 iPhone 11 Pro
  * Firebase SDK version: 6.32-spm-beta
  * Firebase Component: Analytics
  * Component version: 6.32
  * Installation method: `Swift Package Manager`

### [REQUIRED] Step 2: Describe the problem

Unable to run apps on iOS Simulator or device.

```
This app could not be installed at this time.
---
Could not install at this time.
Failed to load Info.plist from bundle at path /Users/harry/Library/Developer/CoreSimulator/Devices/081698CA-BF0E-4B83-894C-09495D52D29A/data/Library/Caches/com.apple.mobile.installd.staging/temp.1DM2Dh/extracted/Payload/AppName.app/Frameworks/GoogleAppMeasurement.framework; Extra info about Info.plist: Couldn't stat /Users/harry/Library/Developer/CoreSimulator/Devices/081698CA-BF0E-4B83-894C-09495D52D29A/data/Library/Caches/com.apple.mobile.installd.staging/temp.1DM2Dh/extracted/Payload/AppName.app/Frameworks/GoogleAppMeasurement.framework/Info.plist: No such file or directory
```

If I tap `Details` it shows the following:

```
This app could not be installed at this time.
Domain: IXUserPresentableErrorDomain
Code: 1
Failure Reason: Could not install at this time.
Recovery Suggestion: Failed to load Info.plist from bundle at path /Users/harry/Library/Developer/CoreSimulator/Devices/081698CA-BF0E-4B83-894C-09495D52D29A/data/Library/Caches/com.apple.mobile.installd.staging/temp.1DM2Dh/extracted/Payload/AppName.app/Frameworks/GoogleAppMeasurement.framework; Extra info about Info.plist: Couldn't stat /Users/harry/Library/Developer/CoreSimulator/Devices/081698CA-BF0E-4B83-894C-09495D52D29A/data/Library/Caches/com.apple.mobile.installd.staging/temp.1DM2Dh/extracted/Payload/AppName.app/Frameworks/GoogleAppMeasurement.framework/Info.plist: No such file or directory
--
Failed to load Info.plist from bundle at path /Users/harry/Library/Developer/CoreSimulator/Devices/081698CA-BF0E-4B83-894C-09495D52D29A/data/Library/Caches/com.apple.mobile.installd.staging/temp.1DM2Dh/extracted/Payload/AppName.app/Frameworks/GoogleAppMeasurement.framework; Extra info about Info.plist: Couldn't stat /Users/harry/Library/Developer/CoreSimulator/Devices/081698CA-BF0E-4B83-894C-09495D52D29A/data/Library/Caches/com.apple.mobile.installd.staging/temp.1DM2Dh/extracted/Payload/AppName.app/Frameworks/GoogleAppMeasurement.framework/Info.plist: No such file or directory
Domain: MIInstallerErrorDomain
Code: 35
User Info: {
    FunctionName = "-[MIBundle _validateWithError:]";
    LegacyErrorString = PackageInspectionFailed;
    SourceFileLine = 131;
}
--


System Information

macOS Version 10.15.6 (Build 19G2021)
Xcode 12.0 (17219)
```

#### Steps to reproduce:

This seems to be a problem with Firebase Analytics because not selecting this when integrating via SPM will result in the app functioning and running as normal. 

To reproduce, try running a project on the latest SPM branch in Xcode 12 and selecting Firebase Analytics during the setup process. 

It might be related to https://github.com/firebase/firebase-ios-sdk/issues/6472 but I am not sure.

**Comment by ryanwilson** [comment]
Hey @harrygt - sorry for the issue you're seeing. This looks like #6360 which we resolved already, see https://github.com/firebase/firebase-ios-sdk/issues/6360#issuecomment-691271702 for more context.

tl;dr: "To pick up the fix - File -> Swift Packages -> Update to Latest Package Versions"

I'll close this issue but if you're on the latest version and still see the exact same issue, please re-open and let us know how you can reproduce it. Thanks!

**Comment by harrygt** [comment]
Thanks Ryan, yes this resolves it!
